save-demo-agent/src/nativeMain/kotlin/com/saveourtool/save/demo/agent/DemoAgentConfig.kt

```diff
@@ -61,5 +59,5 @@ data class RunConfiguration(
  */
 @Serializable
 data class ServerConfiguration(
-    val port: Int = 23456,
+    val port: Long = 23456L,
```

Comment:
This is a TCP port number, right?

But why? `Int` is more than enough.

Reply:
The point is that ktor receives port as `Long`. In order to not to cast it explicitly I've decided to do so.

Reply:
Ah, ok.